### DISCUSSION TITLE
Parsing JSON file

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/PubSec-Info-Assistant#387] [General]
Today, JSON files are treated the same way as TXT:
![image](https://github.com/microsoft/PubSec-Info-Assistant/assets/105317767/4539123e-8b51-4c11-8de5-5e4f970e1775)
 https://github.com/microsoft/PubSec-Info-Assistant/blob/main/functions/FileLayoutParsingOther/__init__.py#L98-L100

Assuming that my JSON file is a list of dictionaries, satisfying the requirement https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/file_utils/filetype.py#L458-L471, would the following allow handling JSON document fields more natively:?

```python
        elif file_extension == '.txt':
            from unstructured.partition.text import partition_text
            elements = partition_text(file=bytes_io)
            
        elif file_extension == '.json':
            from unstructured.partition.json import partition_json
            elements = partition_json(file=bytes_io)
```

**Comment by lon-tierney** [comment]
Thank you for your proposal - we agree that there's more which can be done with JSON. Have you tested this out to see if the code improves? If so, please submit a PR to vNextDev branch.
